### PR TITLE
Makes a bunch of things use playsound_direct

### PR DESCRIPTION
**Pull request by Cyberboss**
Meant to do this ages ago.

Follow-up to #25876 
Closes #26726

### CODE REVIEW

File: code/modules/mob/living/carbon/carbon_defense.dm [hunk 1]
```diff
@@ -294,7 +294,7 @@
 						// you need earmuffs, inacusiate, or replacement
 				else if(ears.ear_damage >= 5)
 					to_chat(src, "<span class='warning'>Your ears start to ring!</span>")
-				src << sound('sound/weapons/flash_ring.ogg',0,1,0,250)
```
Comment (ChangelingRain) [inline code/modules/mob/living/carbon/carbon_defense.dm]: are any of these arguments important

Comment (Cyberboss) [inline code/modules/mob/living/carbon/carbon_defense.dm]: Only the channel arg, volume can't go above 100.

Comment (ChangelingRain) [inline code/modules/mob/living/carbon/carbon_defense.dm]: you should... check that, maybe

Comment (Cyberboss) [inline code/modules/mob/living/carbon/carbon_defense.dm]: http://www.byond.com/docs/ref/info.html#/proc/sound

`volume: 100 for full volume (default), 0 for none, or any value in between `

**Comment by Really-Good-Soda-Flavor** [comment]
I can understand why you would want to get defensive over something you've added, but this is completely pointless. If you set the mob as the source, then there is no need for this duplicate proc.

**Comment by XDTM** [comment]
Some hallucinations use playsound_local with the mob's turf as origin, they could be replaced with direct

File: code/game/machinery/cloning.dm [hunk 2]
```diff
@@ -376,7 +376,7 @@
 		flash_color(occupant, flash_color="#960000", flash_time=100)
 		to_chat(occupant, "<span class='warning'><b>Agony blazes across your consciousness as your body is torn apart.</b><br><i>Is this what dying is like? Yes it is.</i></span>")
 		playsound(src.loc, 'sound/machines/warning-buzzer.ogg', 50, 0)
-		occupant << sound('sound/hallucinations/veryfar_noise.ogg',0,1,50)
+		occupant << sound('sound/hallucinations/veryfar_noise.ogg', 0, 1, 50)
```
Comment (lzimann) [inline code/game/machinery/cloning.dm]: This one is still using `<<`

**Comment by optimumtact** [comment]
Why are there two wrappers for playsound that can be achieved by just setting the sound source variable correctly?

**Comment by Really-Good-Soda-Flavor** [comment]
That's exactly what I am saying.

**Comment by duncathan** [comment]
this proc is redundant

**Pull request closed without merging**